source/guide_typo3-cms.rst

```diff
@@ -0,0 +1,164 @@
+.. author:: Tobias Stahn <info@tstahn.io>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/typo3.png
+      :align: center
+
+#########
+TYPO3 CMS
+#########
+
+`TYPO3 CMS`_ is an Open Source Enterprise Content Management System licensed under `GPL v2`_ and provides the basis for
+more than 500.000 websites, intranets and other web applications worldwide.
+
+First released in 1997 by Kasper Skårhøj, a Danish developer, when the term "Content Management" was still widely
+unknown, it is represented today by the `TYPO3 Association`_ responsible for coordinating and funding the further
+development of the platform.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * MySQL_
+  * composer_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.2:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.2'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+TYPO3 9.5 provides the directory ``public`` as web root, so we will install it next to the default `document root`_ and
+use a symlink to make it accessible.
+
+``cd`` into the directory above your document root ``/var/www/virtual/$USER/`` and set up a new project using composer_.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ composer create-project typo3/cms-base-distribution typo3-cms
+ Installing typo3/cms-base-distribution (v9.5.0)
+   - Installing typo3/cms-base-distribution (v9.5.0)
+ Created project in typo3-cms
+ Loading composer repositories with package information
+ Updating dependencies (including require-dev)
+ Package operations: 76 installs, 0 updates, 0 removals
+   - Installing typo3/cms-composer-installers (v2.2.1)
+   - Installing typo3/class-alias-loader (1.0.1)
+ [...]
+ [isabell@stardust isabell]$
+
+Now remove the `document root`_ and create a symlink to the ``public`` directory.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ rm -rf html
```

Comment:
Even though all guides assume an empty, newly created uberspace, some newer users might not adhere to this principle. What do you think about adding a `.. warning::` above this command block, explaining that everything will be deleted?

Reply:
Better safe than sorry, so fine with me.